python/paddle/fluid/backward.py

```diff
@@ -1253,7 +1256,8 @@ def append_backward(loss,
                       int(core.op_proto_and_checker_maker.OpRole.Loss))
 
     if callbacks is not None:
-        isinstance(callbacks, list)
+        check_type(callbacks, 'callbacks', (list),
```

Comment:
I think you can write `list` or `(list, )` instead of `(list)`

Reply:
Thanks, done